module/modmanager/manager_test.go

```diff
@@ -1425,11 +1413,10 @@ func TestModularDiscovery(t *testing.T) {
 			test.That(t, err, test.ShouldBeNil)
 			t.Logf("Casted struct: %+v", discoveryResult)
 
-			test.That(t, len(discoveryResult.Discovery), test.ShouldEqual, 1)
-			discovery := discoveryResult.Discovery[0]
-			test.That(t, discovery.Query.Subtype, test.ShouldEqual, "rdk:component:generic")
-			test.That(t, discovery.Query.Model, test.ShouldEqual, "rdk:test:helper")
-			test.That(t, discovery.Results["extra"], test.ShouldEqual, tc.expectedExtra)
+			test.That(t, len(discoveryResult), test.ShouldEqual, 1)
```

Comment:
Are we expecting `discoveryResult` to resemble a specific map?

Reply:
Not 100% sure I understand what "specific map" means. Do you mean like a specific `map[string]{x}` x type?

Reply:
We expect it to be a `map[string]interface{}` since we're returning the result of an `asMap` call

Reply:
Hence why in tests we now have to do a string type assertion

Reply:
Is that testing just the extra passed into the Discover Function, or am I missing something about what this test aims to achieve?

Reply:
I just wanted to see that extra is being passed all the way through can be used to alter behavior.